src/queries/pointsOfInterest.js

```diff
@@ -147,3 +149,79 @@ export const GET_POINT_OF_INTEREST = gql`
     }
   }
 `;
+
+// TODO: add newsItems and tours
+// TODO: which data is really necessary in each?
+export const GET_POINT_OF_INTEREST_CROSS_DATA = gql`
```

Comment:
i assumed we would have `newsItems`, `pointsOfInterest`, `tours`, `genericItems` as fields/properties of the POI query. am i misunderstanding the concept here, or why do we have a separate query for this?

Reply:
no, the server provides the corresponding ids and the app needs to query them. this does not work simultaneously. i will send you a link to the slack conversation about it.